packages/core/src/worker/helpers/createSourceObservable.ts

```diff
@@ -74,6 +75,13 @@ NOTE: Only ${pageExtensions.join(
           route: page.route ? page.route.toLowerCase() : page.fullPath.toLowerCase()
         });
       }, [])
-    )
+    ),
+    exponentialBackOffRetryStrategy(schedule),
+    catchError(error => {
+      console.log(
+        `[Mosaic] Source failed and retries are ${schedule.retryEnabled ? 'exhausted' : 'disabled'}`
```

Comment:
will this identify which source became disabled ?

Reply:
Pushed another commit:

Example log:

```
[Mosaic] Source site#91fb4fd4 failed and retries are exhausted
```